e2e/tests/spreadsheet.spec.ts

```
@@ -0,0 +1,483 @@
+    await openSpreadsheetView(page);
+    await expect(page.getByText("Apples").first()).toBeVisible({
+      timeout: 5000,
+    });
+    await expect(page.getByText("Bananas").first()).toBeVisible();
+    await expect(page.locator("text=100").first()).toBeVisible();
+  });
+
+  test("legacy inline sheets render without a fetch", async ({ page }) => {
+    const sheets = [
+      { name: "Inline", data: [[{ v: "Legacy" }, { v: 1 }]] },
+    ];
```

ESLint

[error] Replace `⏎······{·name:·"Inline",·data:·[[{·v:·"Legacy"·},·{·v:·1·}]]·},⏎····` with `{·name:·"Inline",·data:·[[{·v:·"Legacy"·},·{·v:·1·}]]·}`

(prettier/prettier)